http/src/interfaces.ts

```diff
@@ -31,6 +31,7 @@ export interface RequestOptions {
   category?: string;
   lazy?: boolean;
   responseType?: string;
+  ok?: Function;
```

Comment:
I would change that to `(res: SuperAgentResponse) => boolean` to make the typings better

Reply:
Ok I change that. I'm not very familiar with typescript yet.